packages/composites/reference-transform/schemas/env.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/reference-transform-adapter.json",
+  "title": "@chainlink/reference-transform-adapter env var schema",
+  "required": ["SOURCE_DATA_PROVIDER_URL", "RPC_URL"],
+  "type": "object",
+  "properties": {
```

Comment:
```suggestion
  "properties": {
   "RPC_URL": {
      "type": "string",
      "format": "uri"
    },
  
```

Reply:
Actually this might get pulled in by `"$ref": "https://external-adapters.chainlinklabs.com/schemas/ea-reference-data-reader.json"`

Reply:
This is already included since it has json-rpc-adapter as a dep